configure

```diff
@@ -19446,7 +19446,7 @@ ac_config_files="$ac_config_files GNUmakefile src/Makefile.global"
 
 
 ac_config_links="$ac_config_links src/backend/port/pg_sema.c:${SEMA_IMPLEMENTATION} src/backend/port/pg_shmem.c:${SHMEM_IMPLEMENTATION} src/include/pg_config_os.h:src/include/port/${template}.h src/Makefile.port:src/makefiles/Makefile.${template}"
-
+ac_config_links="$ac_config_links src/include/port/wasm_common.h:src/include/port/wasm_common.h"
```

Comment:
try to avoid patching configure file, this file is auto generated whenever upstream update it with autotools and makes it hard for diff to apply. prefer add changes to  CONFIG_SITE  when it is possible